pom.xml

```diff
@@ -198,12 +248,14 @@
         <module>cas-client-integration-jboss</module>
         <module>cas-client-support-distributed-ehcache</module>
         <module>cas-client-support-distributed-memcached</module>
+        <module>cas-client-support-saml</module>
```

Comment:
Is the CommonsUtils JodaTime stuff used for anything beyond SAML?  If not, we should move it to reduce dependencies for the core client